cluster/images/etcd/rollback/rollback.go

```diff
@@ -325,7 +325,8 @@ func applyRequest(r *pb.Request, applyV2 etcdserver.ApplierV2) {
 		applyV2.Put(r)
 	case "DELETE":
 		applyV2.Delete(r)
-	case "POST", "QGET", "SYNC":
+	//case "POST", "QGET", "SYNC":
+	case "POST", "GET", "SYNC":
```

Comment:
Well, I check the the usage of "QGET" in call trace and please revert back this fix.  😞